reference/7.3/Microsoft.PowerShell.Core/About/about_Language_Modes.md

```diff
@@ -14,66 +14,168 @@ Explains language modes and their effect on PowerShell sessions.
+users to have a usable interactive shell experience, running cmdlets and native
+commands, as well as access to basic language elements. But the user can't
+access PowerShell, .NET, or COM APIs that could be abused by a malicious actor.
+
+Any script or script-based module executed in this session runs in
+`ConstrainedLanguage` mode. However, any script or script-based module allowed
+by the policy runs in `FullLanguage` mode without any constraints. This way, a
+system locked down by policy can have scripts that are trusted by the policy
+and run with few constraints.
+
+### Using a session configuration
+
+PowerShell remoting optionally supports creating custom session configurations.
+configuration. You can set the language mode you want for that custom
+configuration. PowerShell Just Enough Administration (JEA) configurations use
+`NoLanguage` mode to restrict sessions to command invocation only. With JEA,
+the remote session can be restricted to specific users. The JEA users are
+limited to running a defined set of commands and can't directly access APIs,
+the file system, or other system resources.
+
+For more information, see [JEA Session configurations][01] and
+[New-PSSessionConfigurationFile][04].
 
-### Language modes
+## Language mode features and limitations
 
 This section describes the language modes in PowerShell sessions.
 
-#### Full language (FullLanguage)
+### FullLanguage mode
 
-The **FullLanguage** mode permits all language elements in the session.
-FullLanguage is the default language mode for default sessions on all versions
-of Windows except for Windows RT.
+The `FullLanguage` mode permits all language elements in the session.
+`FullLanguage` is the default language mode for default sessions on all
+versions of Windows.
 
-#### Restricted language (RestrictedLanguage)
+### RestrictedLanguage mode
 
-In **RestrictedLanguage** mode, users may run commands (cmdlets, functions, CIM
-commands, and workflows) but are not permitted to use script blocks.
+The `RestrictedLanguage` mode was created for Windows RT, which is no longer
+supported. This mode still exists, but it's deprecated and no longer used.
+
+In `RestrictedLanguage` mode, users can run commands (cmdlets, functions, CIM
+commands, and workflows), but can't use script blocks.
 
 Beginning in PowerShell 7.2, the `New-Object` cmdlet is disabled in
-**RestrictedLanguage** mode when system lockdown is configured.
+`RestrictedLanguage` mode when system lockdown is configured.
 
-By default, only the following variables are permitted in RestrictedLanguage
-mode:
+By default, only the following variables are permitted in
+`RestrictedLanguage` mode:
 
 - `$PSCulture`
 - `$PSUICulture`
 - `$True`
 - `$False`
 - `$Null`
 
-Module manifests, which use **RestrictedLanguage** mode, permit the following
-additional variables as well:
+Module manifests that use `RestrictedLanguage` mode allow the following
+additional variables:
```

Comment:
This language change (`manifests, which` -> `manifests that`) implies that this is an opt-in behavior rather than how manifests work - is that correct? Is there a way to limit the change what language mode the manifests are loaded in?

Reply:
Should probably say "modules manifests for modules loaded in `RestrictedLanguage` mode ..."